certified-connectors/Tesseron Invoice/apiDefinition.swagger.json

```diff
@@ -0,0 +1,245 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tesseron ASM Ticket",
+    "description": "This connector allows you to create, edit and search Tickets in your Tesseron ASM Instance.",
+    "contact": {
+      "name": "Tesseron ASM Support",
+      "email": "development@llnet.de",
+      "url": "https://www.docs.tesseron.de"
+    },
+    "version": "1.0"
+  },
+  "host": "dynamichosturl",
+  "basePath": "/ApiInvoice",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [],
+  "paths": {
+    "/CreateInvoicePositionNote": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default"
+          },
+          "200": {
+            "description": "success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Message": {
+                  "type": "string",
+                  "description": "Message"
+                },
+                "Success": {
+                  "type": "boolean",
+                  "description": "Success"
+                },
+                "invoiceNoteId": {
+                  "type": "string",
+                  "description": "Invoice Note ID"
+                }
+              }
+            }
+          }
+        }
+      },
+      "summary": "Create Position Note",
```

Comment:
@developmentLL the operation properties should be within the scope of post action. before or after the responses.
![image](https://user-images.githubusercontent.com/97551789/217327246-6c421983-b7d9-413e-9a88-c70a252ba1ce.png).

Please fix all operations in the swagger file.